boards/nordic/nrf54l15dk/doc/index.rst

```diff
@@ -0,0 +1,138 @@
+.. _nrf54l15dk_nrf54l15:
+
+nRF54L15 DK
+############
+
+Overview
+********
+
+.. note::
+
+   All software for the nRF54L15 SoC is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
+
+The nRF54L15 Development Kit hardware provides support for the Nordic Semiconductor
+nRF54L15 Arm Cortex-M33 CPU and the following devices:
+
+* :abbr:`SAADC (Successive Approximation Analog to Digital Converter)`
+* CLOCK
+* RRAM
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`TWIM (I2C-compatible two-wire interface master with EasyDMA)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`GRTC (Global real-time counter)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/nrf54l15dk_nrf54l15.webp
+     :align: center
+     :alt: nRF54L15 DK
+
+     nRF54L15 DK (Credit: Nordic Semiconductor)
+
+Hardware
+********
+
+nRF54L15 DK has two crystal oscillators:
+
+* High-frequency 32 MHz crystal oscillator (HFXO)
+* Low-frequency 32.768 kHz crystal oscillator (LFXO)
+
+The crystal oscillators can be configured to use either
+internal or external capacitors.
+
+Supported Features
+==================
+
+The ``nrf54l15dk/nrf54l15/cpuapp`` board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| SAADC     | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| RRAM      | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| TWIM      | on-chip    | i2c                  |
```

Comment:
no twis support?

Reply:
Only on nrfx level. No implementation in i2c shim